src/Form/UserSubscriptionsForm.php

```diff
@@ -161,7 +182,9 @@ protected function buildFlagList(): array {
 
     if (empty($results)) {
       $build['no_results'] = [
-        '#plain_text' => $this->t('No subscriptions found.'),
+        '#type' => 'html_tag',
```

Comment:
This is no good. If we need html, we need a template.

Reply:
Added